modules/default.nix

```diff
@@ -437,6 +453,18 @@ in {
         type = types.bool;
         default = true;
       };
+      bundlerGemGroups = lib.mkOption {
```

Comment:
if you can think of a better name for this option, please suggest it. This is just a proof of concept to implement it but there may be nicer options than what I've done here...